src/ODM/MongoDB/Types/CurrencyType.php

```diff
@@ -2,10 +2,10 @@
 
 namespace ZFBrasil\DoctrineMoneyModule\ODM\MongoDB\Types;
 
-use Doctrine\ODM\MongoDB\Types\StringType;
+use Doctrine\ODM\MongoDB\Types\Type;
 use Money\Currency;
 
-class CurrencyType extends StringType
+class CurrencyType extends Type
```

Comment:
almost all data mapped to string on database side, is more interesting PHP side. because we speak from ODM (ORM) here. Look at times http://doctrine-dbal.readthedocs.org/en/lates/reference/types.html#custom-mapping-types  for u should be a float type :). (may better way ObjectType -> serializing of money inclusive currency as one field)